app/controllers/pics_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PicsController < ApplicationController
+
+
+
+  def show
+    @pic = Pic.find(params[:id])
+  end
+
+  def new
+
+  end
+
+
+  # def upload
+  #   uploaded_file = params[:photo]
+  #     File.open(Rails.root.join('public', 'uploads', uploaded_file.original_filename), 'wb') do |file|
+  #     file.write(uploaded_file.read)
+  #   end
+  # end
+
+  def create
+    @pic = Pic.new(article_params)
+
+    @pic.save
```

Comment:
Could add an if/else statement for occasions when the save fails, maybe notifying the user with an alert